packages/@aws-cdk/aws-codedeploy/lib/lambda/custom-deployment-config.ts

```diff
@@ -0,0 +1,151 @@
+import { Construct, Duration } from '@aws-cdk/core';
```

Comment:
I believe we changed this to use the `Construct` class from the `constructs` library. Should be apparent when you update from `origin/master`.

Reply:
I did the update locally and started getting errors like this, when checking if a stack has a resource. Do you know what this is about?

`
TypeError: Cannot read property 'children' of undefined
    at synthNestedAssemblies (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/core/lib/private/synthesis.ts:50:33)
    at synthNestedAssemblies (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/core/lib/private/synthesis.ts:54:7)
    at synthNestedAssemblies (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/core/lib/private/synthesis.ts:54:7)
    at Object.synthesize (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/core/lib/private/synthesis.ts:14:3)
    at App.synth (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/core/lib/stage.ts:174:23)
    at synthesizeApp (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/assert/lib/synth-utils.ts:76:15)
    at Function._synthesizeWithNested (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/assert/lib/synth-utils.ts:53:22)
    at Object.expect (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/assert/lib/expect.ts:9:45)
    at Object.dependency on the config exists to ensure ordering (/home/ec2-user/environment/aws-cdk/packages/@aws-cdk/aws-codedeploy/test/lambda/test.custom-deployment-config.ts:226:5)
    at Object.<anonymous> (/home/ec2-user/environment/aws-cdk/node_modules/nodeunit/lib/core.js:236:16)
`

Reply:
Never mind, I was able to work past this. I can't extend construct directly.